index.ln

```diff
@@ -1 +1,9 @@
-export const comeGetMe = "You got me!"
+from @std/app import start, exit
```

Comment:
This import line doesn't need to be here for this file. But if "nestedhellodep" uses it in `echo` then it still makes sense to be in the `.dependencies.ln` file.